src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.bookstore.controller;
+
+import java.util.List;
+import mate.academy.bookstore.dto.BookDto;
+import mate.academy.bookstore.dto.CreateBookRequestDto;
+import mate.academy.bookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
move project-wide prefix `/api` to `server.servlet.context-path`

Reply:
not fixed